### PR TITLE
khanhvnn - Exercise Week 6: Develop test script for below functions of Opencart web application

### PR DESCRIPTION
**Pull request by khanhvnv**
## Exercise

* [Exercise-week6](https://github.com/NT-Line-team/Behave_test/issues/49)

## Overview

Develop test script for below functions of Opencart web application.
```
Search a product (finished)
```
## Verification Steps

1. `git checkout pr/###`
2. Check the functionality: 
- Run the files with below commands.
python ./search_product.py
(change the variables (e,g: port, search_phone.. where neccessary)
### Docs
- [ ] No Doc Changes.

### CODE REVIEW

File: exercises/khanhvnv/search_product.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+from selenium.webdriver.support.select import By
+from hamcrest import assert_that, contains_string, equal_to
+
+#Variables
+url = 'http://localhost:88/shop'  # Load page
+SEARCH_BOX = (By.NAME, 'search')
+SEARCH_RESULTS = (By.CSS_SELECTOR, 'div.image > a > img.img-responsive')
+search_key1 = 'Phone'
+search_key2 = 'No Results'
+
```
Comment (hungphamviet) [inline exercises/khanhvnv/search_product.py]: please capitalize all characters in the constants `url`, `search_key1`, `search_key2`. Also, please name the constants to be more meaningful. For example,

```
SEARCH_PHONE = 'Phone'
SEARCH_NO_RESULTS = 'No Results'
```

**Comment by khanhvnv** [comment]
Hi Hung,

Many thanks for your feedback. 
I have updated the search_product.py follow your comments and add 2 more files: review_product.py and shopping_checkout.py.
Please help me to review them. And so sorry for submitting the exercises late.
Best Regards,
Khanh

**Comment by khanhvnv** [comment]
Hi @hungphamviet,

Many thanks for your feedback.
I have created new PR  'khanhvnn - Exercise Week 6_part 2' for the new files: https://github.com/NT-Line-team/Behave_test/pull/81 

Please help me to review them also. 

Best Regards,
Khanh

**Comment by hungphamviet** [comment]
Merged #74 into `develop`

**Pull request merged**